src/framework/preferences/messages/letterSpace_fa.json

```diff
@@ -0,0 +1,6 @@
+{
+    "label": "",
+    "description": "",
+    "increaseLabel": "",
+    "decreaseLabel": ""
+}
```

Comment:
It seems that the translations for this adjuster are missing.

Reply:
I spoke with Sepi about this one, and while I initially thought it was an oversight, I'd done it intentionally. It turns out that the idea of letter spacing in Farsi doesn't really apply, and I didn't want it to fall back to another localization, so I added the blank strings with the anticipation that we would be working on the ability to activate/deactivate certain preferences by localization at some point (I believe there is a Jira for this but I can't find it at the moment). I wanted to add a comment in the file but Atom didn't like it, so I didn't.

What do you think is the best solution for this issue?

Reply:
It's a good question. I think it's probably better to have the localized text, otherwise the UI will just look broken. And the adjustments would still apply if they tried to use it. Probably better that they know what it does. Also, if originally they set the letter spacing for a different language at least they'd know where to go to change it back. That's just my thoughts.. I think long term we'd also need to know what languages not to apply settings too, as the letter spacing could already be part of their prefs set.